samples/android-app/build.gradle.kts

```diff
@@ -10,19 +10,26 @@ apply {
 
 configure<AppExtension> {
     compileSdkVersion(27)
+
     defaultConfig {
         minSdkVersion(23)
         targetSdkVersion(27)
         versionCode = 1
         versionName = "1.0"
         testInstrumentationRunner = "android.support.test.runner.AndroidJUnitRunner"
     }
+
     buildTypes {
         getByName("release") {
             isMinifyEnabled = false
             proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
         }
     }
+
+    compileOptions {
+        setSourceCompatibility(JavaVersion.VERSION_1_8)
+        setTargetCompatibility(JavaVersion.VERSION_1_8)
+    }
```

Comment:
:x: Should the changes in this file be part of this PR?

Reply:
@Tapchicoma Can you explain why this was necessary. I'm fine with tossing in things that are unrelated as long as they are different commits and you note that the PR shouldn't be squashed.

Reply:
It is not actually related to plugin, but related to deprecated warning when running plugin for samples. The warning was:

Reply:
Fair enough.